conda_smithy/ci_register.py

```diff
@@ -178,48 +178,38 @@ def add_project_to_travis(user, project):
     headers = travis_headers()
     endpoint = 'https://api.travis-ci.org'
 
-    url = '{}/hooks'.format(endpoint)
+    url = '{}/{}/{}/key'.format(endpoint, user, project)
 
     found = False
     count = 0
+    response = requests.get(url, headers=headers)
+    content = response.json()
 
-    while not found:
```

Comment:
Because if there was an error thrown, it was re-raised and therefore the while loop was unnecessary there